CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Improvements
 
 - provide folding for list of sibling attributes by @apupier (#719)
+- support remote includes when exporting in PDF (#731)
```

Comment:
Since we can add additional command line arguments, I guess we should say: "by default, allow remote includes when exporting in PDF"?

Reply:
with current implementation, it is not only by default. The parameter is always provided. The parameter from the settings are appended, they are not replacing some parameters that are always passed.

Reply:
Sorry I wasn't clear.
My point was that we already support remote includes when exporting in PDF since we can set `-a allow-uri-read` in the extension settings. I was suggesting to reword the changelog to mention that remote includes are now enabled by default when exporting to PDF.